src/amber/server/server.cr

```diff
@@ -74,8 +74,14 @@ module Amber
           server.listen(settings.port_reuse)
           exit
         rescue e : Errno
-          logger.error e.message
-          logger.info "Restarting server..."
+          if e.message == "accept: Too many open files"
```

Comment:
What is the error type in this case? Is the error type too generic to just rescue the this type of errors?

Reply:
The error type is the one I'm rescuing from. Errno

Reply:
Errno is the error returned for all errors related to creating a tcp socket.